src/components/login/container/index.jsx

```diff
@@ -57,58 +59,47 @@ export class Login extends Component {
               expert-defined content with your own custom concepts.
             </h6>
             <form className="form-wrapper" onSubmit={this.handleLogin}>
-              <fieldset>
-                <legend>Log In to Open Concept Lab</legend>
-                <div className="form-group">
-                  <label htmlFor="username">Username</label>
-                  <input
-                    type="text"
-                    className="form-control"
-                    placeholder="OCL Username"
-                    name="username"
-                    id="username"
-                    required
-                    value={username.trim()}
-                    onChange={this.handleInput}
-                  />
```

Comment:
Do all these changes have anything to do with the ticket?

Reply:
The same here. Let me revert these unrelated changes.